src/help-widget-renderer.ts

```diff
@@ -35,13 +35,16 @@ export class BBHelpHelpWidgetRenderer {
     return invoker;
   }
 
-  public createMenu(contentUrl: string): HTMLDivElement {
+  public createMenu(contentUrl: string, newFeatureConfig: WhatsNewConfig): HTMLDivElement {
     const menuItemInfo: MenuItem[] = [
-      { label: 'Open help', url: contentUrl, class: 'bb-help-content-link' },
-      { label: 'What\'s new', url: 'https://duckduckgo.com' },
+      { label: 'Open help', url: contentUrl, class: 'bb-help-content-link', newTab: true },
       { label: SEPARATOR },
```

Comment:
Is this a placeholder to add What's new back to the menu? It looks like these code changes are removing the What's new item, but maybe I'm missing something.